### PR TITLE
Azure stack: Add Samples under Hybrid

### PR DESCRIPTION
**Pull request by viananth**

### CODE REVIEW

File: Hybrid/README.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
+---
+services: azure-resource-manager
+platforms: ruby
+author: devigned
```
Comment (bganapa) [inline Hybrid/README.md]: credit to yourself :) not to david

File: Hybrid/README.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+contains a single subnet with a single network security group rule which allows traffic on port 22 for ssh with a single
+network interface belonging to the subnet. The virtual machine is a `Standard_A1` size. You can find the template
+[here][Template].
+
+### To run this sample, do the following:
+
+You will need to create an Azure service principal either through Azure CLI, PowerShell or the portal. You should gather
+each the Tenant Id, Client Id and Client Secret from creating the Service Principal for use below.
+
+- [Create a Service Principal][ServicePrincipalCreation]
```
Comment (bganapa) [inline Hybrid/README.md]: Is there a sample on how to create a service principal through Ruby SDK? I have seen PS/Portal/CLI examples..

File: Hybrid/lib/deployer.rb [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+        active_directory_settings
+        )
+
+    credentials = MsRest::TokenCredentials.new(provider)
+
+    options = {
+      credentials: credentials,
+      subscription_id: @subscription_id,
+      active_directory_settings: active_directory_settings,
+      base_url: 'https://management.local.azurestack.external' # Your Resource Manager Url
```
Comment (bganapa) [inline Hybrid/lib/deployer.rb]: I think Arm endpoint could be a ENV var.

File: Hybrid/README.md (hunk 2 above)
Comment (sarangan12) [reply Hybrid/README.md]: I dont think so

**Pull request merged**